walkthroughs/week-5-tdd/project/src/test/java/com/google/sps/FindMeetingQueryTest.java

```diff
@@ -270,5 +270,110 @@ public void notEnoughRoom() {
 
     Assert.assertEquals(expected, actual);
   }
-}
 
+
+
+    @Test
+  public void everyAttendeeIsConsideredWithAllDayC() {
```

Comment:
What is "AllDayC"? In general, it is better not to use abbreviations or acronyms in code.